verification/verify.go

```diff
@@ -146,9 +150,9 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 	}
 	sv.count++
 
-	// Append retryable errors to results
+	// Append non-fatal retryable errors to results
 	// The caller should terminate processing for non-retryable errors
-	if IsRetryable(err) {
+	if !IsFatal(err) && IsRetryable(err) {
```

Comment:
So we're only defining `ErrAudioMismatch` as a fatal error now which allows results that trigger `ErrPixelMismatch` to be appended to `sv.results` - ok, that works for now. 

We eventually might need an error type that describes both `ErrAudioMismatch` *and* `ErrPixelMismatch` - this error type would indicate that the orchestrator definitely did something wrong (the broadcaster would not re-add these orchestrators to its working set - see [this comment](https://github.com/livepeer/go-livepeer/pull/1384/files#r382798632)). But, we can address that outside of this PR when we actually handle re-adding orchestrators back to the working set.